### PR TITLE
Changed block size for 256kb fio load tests to 16kb

### PR DESCRIPTION
**Pull request by abhinav2603**
Changed block size for 256kb fio load tests to 16kb

### CODE REVIEW

**Review by vadlakondaswetha — APPROVED** [review]

File: perfmetrics/scripts/job_files/seq_rand_read_write.fio [hunk 1]
```diff
@@ -21,15 +21,15 @@ filename_format=$jobname.$jobnum.$filenum
...
+bs=16k
 directory=gcs/256kb
 filesize=256k
 numjobs=40
 
 [2_thread]
 stonewall
 startdelay=670
-bs=256k
+bs=16k
```
Comment (vadlakondaswetha) [inline perfmetrics/scripts/job_files/seq_rand_read_write.fio:32]: is this change intented

File: perfmetrics/scripts/utils/metrics_util_test.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
+"""Test for fio output logging."""
+
+import unittest
+import os
+from metrics_util import remove_old_log_files
+
+
+class TestMetricsUtil(unittest.TestCase):
+
+  def test_remove_old_log_files(self):
```
Comment (vadlakondaswetha) [inline perfmetrics/scripts/utils/metrics_util_test.py:10]: Please add test cases for these scenarios
dir contains 5 files and we pass 6
dir contains 0 files and we pass >1

File: perfmetrics/scripts/utils/metrics_util.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+Usage:
+python3 metrics_util.py $PATH_TO_LOG_DIR
+"""
+
+import os
+import typing
+import sys
+
+
+def remove_old_log_files(logging_dir: str, num_files_retain: int):
```
Comment (vadlakondaswetha) [inline perfmetrics/scripts/utils/metrics_util.py:12]: rename to remove_old_files

Comment (abhinav2603) [reply perfmetrics/scripts/utils/metrics_util.py]: Done

**Review by vadlakondaswetha — APPROVED** [review]

**Pull request merged**